descriptors/ModuleDescriptor-template.json

```diff
@@ -91,7 +91,9 @@
             "circulation-item-storage.items.item.post",
             "circulation-item-storage.items.item.get",
             "inventory-storage.material-types.collection.get",
-            "inventory-storage.loan-types.collection.get"
+            "inventory-storage.loan-types.collection.get",
```

Comment:
I think we may need the below permissions as well
**circulation-storage.requests.item.get**
**circulation.requests.item.get**
Please check